site/docs/guide.md

```diff
@@ -776,8 +776,17 @@ before the command (`build`, `test`, etc).
 4.  **The user-specified RC file**, if specified with
     <code>--bazelrc=<var>file</var></code>
 
-    This flag is optional. However, if the flag is specified, then the file must
-    exist.
```

Comment:
removed `if the flag is specified, then the file must exist` portion, because bazel would error clearly on non-existent bazelrc file.